src/ngx_http_modsecurity_module.c

```diff
@@ -664,12 +665,24 @@ ngx_http_modsecurity_create_conf(ngx_conf_t *cf)
      *     conf->enable = 0;
      *     conf->sanity_checks_enabled = 0;
      *     conf->rules_set = NULL;
+     *     conf->audit_log_refresh = NULL;
      *     conf->pool = NULL;
      *     conf->transaction_id = NULL;
      */
 
     conf->enable = NGX_CONF_UNSET;
     conf->rules_set = msc_create_rules_set();
+
+    //ngx_str_t log_refresh_file = ngx_string("/dev/null");
+    ngx_str_t log_refresh_file = ngx_string("/usr/local/nginx/brandonull");
```

Comment:
Temporary change so it is easy to use `lsof` and distinguish between other nignx things that opened `/dev/null`.